index.html

```diff
@@ -0,0 +1,165 @@
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="./style.css">
+    <title>Capstone Project</title>
+</head>
+<body>
+    <header>
+      <nav>
+        <div class="bt">
+          <ul class="nav-menu1">
+            <li class="nav-item1">
+              <a class="nav-link1" href="#about"><img src="./Images/twitter (1).png" alt="twitter icon"></a>
+            </li>
+            <li class="nav-item1">
+              <a class="nav-link1" href="#contact"><img src="./Images/facebook.png" alt="facebook icon"></a>
+            </li>
+          </li>
+            <li class="nav-item1 >                                                                        ">
+              <a class="nav-link1" href="#my-projects">Portfolio</a>
+            </li>
+            <li class="nav-item1">
+              <a class="nav-link1" href="#about">About</a>
+            </li>
+            <li class="nav-item1">
+              <a class="nav-link1" href="#contact">Contact</a>
+              
+          </ul>
+        </div>
+      </nav>
+      </div>
+        <nav class="navbar">
+          
+          <ul class="nav-menu">
+            <li class="nav-item">
+              <a  href="about.html" class="link3">About</a>
+            </li>
+            <li class="nav-item">
+              <a  href="#main"  class="nav-link" id="home">Main</a>
+            </li>
+            <li class="nav-item">
+              <a href="#speakers" class="nav-link">Speaker</a>
+            </li>
+            <li class="nav-item">
+              <a href="#partner" class="nav-link">ENROLl</a>
+            </li>
+            <li class="nav-item">
+              <a href="#footer" class="nav-link2" id="link1">Events</a>
+            </li>
+          </ul>
+          <div class="hamburgar">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+         <a href="about.html">
+           <img class="m-logo" src="./Images/143849_ted_icon.png" alt="ted icon">
+         </a>
+    </header>
+  <section class="btn">
+    <h3 class="l-h3">'Hello World'</h3>
+     <h1 class="ted">In the spirit of TED’s mission, “ideas worth spreading,”</h1>
+       <article class="info">the TEDx program helps communities, organizations and individuals produce TED-style events at the local level. TEDx events are planned and coordinated independently, on a community-by-community basis, under a free license from TED.</article>
+      <h3 class="date1">2023.10.15(BGD) ~ 16(IQ)</h3>
+       <p class="date">@ National Museum of Iraq, In Center of Hall</p>
+  </section>
```

Comment:
- [ ] Kindly make sure to align the `h1` and paragraph of this section to the left, also kindly consider following the font size, font weight... from the provided design

|Your website|Provided design|
|-|-|
|![Screenshot_20230708_115858](https://github.com/newtonah0/Microverse-Capstone-1/assets/25227022/810841e6-be87-4ad0-acf5-6ba02b0aa722)|![Screenshot_20230708_120327](https://github.com/newtonah0/Microverse-Capstone-1/assets/25227022/bb0c4d2e-22a0-40fa-a2cf-0c39142e7446)|

Reply:
- [ ] As mentioned by the previous reviewer, kindly stick to the provided design as much as possible. The title of your website is a little different compared to the provided design. Kindly note that the title is uppercase and the `font-weight: bold`

> * [ ]  Kindly make sure to align the `h1` and paragraph of this section to the left, also kindly consider following the font size, font weight... from the provided design
> 
> Your website	Provided design
> ![Screenshot_20230708_115858](https://user-images.githubusercontent.com/25227022/251968844-810841e6-be87-4ad0-acf5-6ba02b0aa722.png)	![Screenshot_20230708_120327](https://user-images.githubusercontent.com/25227022/251968938-bb0c4d2e-22a0-40fa-a2cf-0c39142e7446.png)